### PR TITLE
Add support for completion when using external browser login

### PR DESCRIPTION
**Pull request by frontegg-david**

### CODE REVIEW

File: android/src/main/java/com/frontegg/android/AuthenticationActivity.kt [hunk 1]
```diff
@@ -103,13 +120,19 @@ class AuthenticationActivity : Activity() {
         private const val AUTH_LAUNCHED = "com.frontegg.android.AUTH_LAUNCHED"
         private const val CUSTOM_TAB_LAUNCHED = "com.frontegg.android.CUSTOM_TAB_LAUNCHED"
         private val TAG = AuthenticationActivity::class.java.simpleName
+        var onAuthFinishedCallback: (() -> Unit)? = null // Store callback
```
Comment (alex-minaiev-frontegg) [inline android/src/main/java/com/frontegg/android/AuthenticationActivity.kt:123]: Did you use this callback? You never fill this property only a safe call

File: android/src/main/java/com/frontegg/android/AuthenticationActivity.kt [hunk 2]
```diff
@@ -103,13 +120,19 @@ class AuthenticationActivity : Activity() {
...
+            activity: Activity,
+            loginHint: String? = null,
+            callback: (() -> Unit)? = null
+        ) {
             val intent = Intent(activity, AuthenticationActivity::class.java)
             val authorizeUri = AuthorizeUrlGenerator().generate(loginHint = loginHint)
             intent.putExtra(AUTH_LAUNCHED, true)
             intent.putExtra(AUTHORIZE_URI, authorizeUri.first)
             intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
+            EmbeddedAuthActivity.onAuthFinishedCallback = callback
```
Comment (alex-minaiev-frontegg) [inline android/src/main/java/com/frontegg/android/AuthenticationActivity.kt:135]: Maybe here you should fill `AuthenticationActivity`.`onAuthFinishedCallback`
Not `EmbeddedAuthActivity`.`onAuthFinishedCallback`

File: android/src/main/java/com/frontegg/android/AuthenticationActivity.kt (hunk 1 above)
Comment (frontegg-david) [reply android/src/main/java/com/frontegg/android/AuthenticationActivity.kt:123]: invoked in the invokeAuthFinished

File: android/src/main/java/com/frontegg/android/AuthenticationActivity.kt (hunk 2 above)
Comment (frontegg-david) [reply android/src/main/java/com/frontegg/android/AuthenticationActivity.kt]: fixed

**Review by dianaKhortiuk-frontegg — APPROVED** [review]

**Review by alex-minaiev-frontegg — APPROVED** [review]

**Pull request merged**